src/NodaTime/LocalDate.cs

```diff
@@ -276,7 +263,7 @@ public static LocalDate FromYearMonthWeekAndDay(int year, int month, int occurre
             Preconditions.CheckArgumentRange(nameof(dayOfWeek), (int) dayOfWeek, 1, 7);
 
             // Correct day of week, 1st week of month.
-            int week1Day = (int) dayOfWeek - startOfMonth.DayOfWeek + 1;
+            int week1Day = dayOfWeek - startOfMonth.DayOfWeek + 1;
```

Comment:
For any enum `Foo`, there's a unary `-` operator as `int -(Foo x, Foo y)` (assuming `int` is the underlying type).